libs/expo/betterangels/src/lib/hooks/californiaIdUniqueCheck/useCaliforniaIdUniqueCheck.ts

```diff
@@ -0,0 +1,25 @@
+import { Regex } from '@monorepo/expo/shared/static';
+import { useClientCaliforniaUniqueCheckQuery } from './__generated__/ClientCaliforniaUniqueCheck.generated';
+
+export default function useCaliforniaIdUniqueCheck(
+  californiaId: string,
+  clientProfileId?: string
+) {
+  const { data } = useClientCaliforniaUniqueCheckQuery({
+    skip: !californiaId || !Regex.californiaId.test(californiaId),
```

Comment:
just the regex would be enough: `skip: !Regex.californiaId.test(californiaId)`

Reply:
so it would